adb-vagrantfile-tests/k8s-tests/README.md

```diff
@@ -0,0 +1,33 @@
+### Current flow
+
+
```

Comment:
yeah, basically provide more detail about it. Not required ATM tough, but keep a note once we have minimal viable up.